mevshare/database.go

```diff
@@ -34,21 +34,41 @@ type DBSbundle struct {
 	SimProfit          sql.NullString `db:"sim_profit"`
 	SimRefundableValue sql.NullString `db:"sim_refundable_value"`
 	SimGasUsed         sql.NullInt64  `db:"sim_gas_used"`
-	Body               []byte         `db:"body"`
-	BodySize           int            `db:"body_size"`
-	OriginID           sql.NullString `db:"origin_id"`
+	// sum of all simulations gas used
+	SimAllSimsGasUsed sql.NullInt64 `db:"sim_all_sims_gas_used"`
```

Comment:
suggestion (non-blocking): We should probably update the notion doc with the new schema fields so data team is aware

cc: @elenahoo @ababino @sui414